### PR TITLE
Added Google Calendar link to email

### PR DESCRIPTION
**Pull request by Kipjr**
created formatting for ISO and edit template email
after #19

### CODE REVIEW

**Comment by SubSide** [comment]
Remove last commit (about updating to latest laravel) as you made a separate pull request for it, and as it isn't described in the pull-request

**Comment by Kipjr** [comment]
That shouldn't have happened

**Comment by Kipjr** [comment]
I guess that is fixed

**Comment by SubSide** [comment]
Have you checked the comment I placed at confirmed.blade.php?

**Comment by Kipjr** [comment]
> Have you checked the comment I placed at confirmed.blade.php?

Where?

File: resources/views/emails/signin/confirmed.blade.php [hunk 1]
```diff
@@ -67,7 +67,6 @@
 Met vriendelijke groet,<br />
 De mensacomputer
 
-Vragen of problemen? Bel de bar: <a href="tel://{{ str_replace('-', '', config('mensa.contact.bar')) }}">{{ config('mensa.contact.bar') }}</a>
-@endcomponent
-
-
+Vragen of problemen? Bel de bar: <a href="tel://{{ str_replace('-', '', config('mensa.contact.bar')) }}">{{ config('mensa.contact.bar') }}</a><br>
+<a href="https://calendar.google.com/calendar/r/eventedit?text=Mensa&dates={{ formatDateISO8601($mensaUser->mensa->date,$mensaUser->mensa->closingTime()) }}&details=Mijn+inschrijving:+{{ route('signin.edit', [$mensaUser->confirmation_code]) }}&Location=Studentenvereniging+V.S.L.+Catena,+Kolfmakersteeg 8,+2311+VG+Leiden,+Netherlands">Voeg toe aan je eigen agenda</a>
```
[the remark this answers is not in the document]
Comment (SubSide) [reply resources/views/emails/signin/confirmed.blade.php:71]: I posted this like 5 days ago, but I see it just posted it a minute ago. I now understand why you haven't seen it yet 🙈

[the remark this answers is not in the document]
Comment (Kipjr) [reply resources/views/emails/signin/confirmed.blade.php:71]: Mucho confusing

[the remark this answers is not in the document]
Comment (Kipjr) [reply resources/views/emails/signin/confirmed.blade.php:71]: How about this? https://gist.github.com/jakebellacera/635416/3c81643cc236a5efdf535fcbf3f876eaaa6c4787

[the remark this answers is not in the document]
Comment (SubSide) [reply resources/views/emails/signin/confirmed.blade.php:71]: Then I guess we have to check how we can include files with mails

[the remark this answers is not in the document]
Comment (SubSide) [reply resources/views/emails/signin/confirmed.blade.php:71]: https://laravel.com/docs/7.x/mail#attachments

And then the raw data attachments I guess

[the remark this answers is not in the document]
Comment (Kipjr) [reply resources/views/emails/signin/confirmed.blade.php:71]: Easy fix would be this:

https://www.google.com/calendar/render?action=TEMPLATE&text=Mensa&details=Menu&location=Catena&dates=20200930T170000Z%2F20200930T183000Z

**Pull request closed without merging**